doc/releases/changelog-dev.md

```diff
@@ -4,9 +4,99 @@
 
 <h3>New features since last release</h3>
 
+<h4>Efficient state preparation for tensor network simulations 🦾</h4>
+
 * Added new ``MPSPrep`` template to prepare quantum states in tensor simulators.
   [(#6431)](https://github.com/PennyLaneAI/pennylane/pull/6431)
```

Comment:
Two thoughts:
1. MPSPrep only works with lightning.tensor. 
2. Lightning.tensor only works with GPU.
With that in mind, do we still want this one to be the first?

Additionally, we have `qml.Superposition` which is also an efficient state prep. We could group them together.

Reply:
It's pull request 6670. Not sure why it's not in the changelog.

Reply:
Thanks Diego this helps! We can definitely add superposition to this (indeed, 6670 had no changelog entry 😢). Do you still think this feature should be further down the list then?

Reply:
I added superposition right below this entry

Reply:
>Do you still think this feature should be further down the list then?

No, I think with the two state prep methods in, it makes more sense to have at the start :+1: